telco-hub/configuration/README.md

```diff
@@ -0,0 +1,218 @@
+```
+
+Wait the operator to be installed:
+
+```bash
+> oc -n openshift-gitops-operator get subscriptions.operators.coreos.com openshift-gitops-operator -o jsonpath='{.status.state}'
+
+AtLatestKnown
+
+> oc -n openshift-gitops-operator get pod
+NAME                                                         READY   STATUS    RESTARTS   AGE
+openshift-gitops-operator-controller-manager-d97fddc-9zmrn   2/2     Running   0          21m
+
+> oc -n openshift-gitops get pod
+NAME                                                          READY   STATUS    RESTARTS   AGE
+cluster-7b65f74f8f-sbx24                                      1/1     Running   0          37s
+gitops-plugin-7d8b6d777b-5npgj                                1/1     Running   0          37s
+kam-7bc6f69fcd-jrtgv                                          1/1     Running   0          37s
+openshift-gitops-application-controller-0                     1/1     Running   0          35s
+openshift-gitops-applicationset-controller-5cddb476fc-q5shw   1/1     Running   0          35s
+openshift-gitops-dex-server-954f978c9-2lp44                   1/1     Running   0          35s
+openshift-gitops-redis-7ff87f9b48-8ld9x                       1/1     Running   0          35s
+openshift-gitops-repo-server-6ccffb9695-pc8bj                 1/1     Running   0          35s
+openshift-gitops-server-845d6798-9c5tv                        1/1     Running   0          35s
+```
+
+## Tune your own overlay layer
+
+Before creating the Telco Hub ArgoCD Application, you have to select the different optional component, and configure all of them. This is done using the different available kustomize patches. 
+
+At this point, in general, you will fork this repo to tune the different kustomize patches and to select the optional components. There exists a `kustomize.yaml` with all the information:
+
+```yaml
+---
+# kustomization file including different overlays over the
+# reference crs
+---
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+resources:
+  # comment the optional components when not using them
+  - reference-crs/optional/lso/
+  - reference-crs/optional/odf-internal/
+  # everything under required is mandatory
+  - reference-crs/required/gitops/
+  - reference-crs/required/acm/
+  - reference-crs/required/talm/
+  # but, include this content if you want to include the argocd
+  # configuration and apps for gitops ztp management of cluster
+  # installation and configuration
+  # - reference-crs/required/gitops/ztp-installation
+
+# following the different overlays to patch
+# the different configurations. In case you are not using some of the
+# optional components, comment the proper patches
+#
+patches:
+  # if you use LocalStorage operator, edit and configure the patch
+  - target:
+      group: local.storage.openshift.io
+      version: v1
+      kind: LocalVolume
+      name: local-disks
+    path: example-overlays-config/lso/local-storage-disks-patch.yaml
+```
```

Comment:
This is now automatically handled by your top level kustomization.yaml. Maybe remove these lines from the README and just point to the included top level kustomization.yaml so that we don't have duplication (this section is already obsolete by the latest)

Reply:
you are right, I had not update the doc about this chage. Now it is done.